web/themes/new_weather_theme/templates/block/block--system-branding-block.html.twig

```diff
@@ -1,35 +1,35 @@
 {% extends "block.html.twig" %}
 {#
 /**
- * @file
- * Theme override for a branding block.
- *
- * Each branding element variable (logo, name, slogan) is only available if
- * enabled in the block configuration.
- *
- * Available variables:
- * - site_logo: Logo for site as defined in Appearance or theme settings.
- * - site_name: Name for site as defined in Site information settings.
- * - site_slogan: Slogan for site as defined in Site information settings.
- */
+* @file
+* Theme override for a branding block.
+*
+* Each branding element variable (logo, name, slogan) is only available if
+* enabled in the block configuration.
+*
+* Available variables:
+* - site_logo: Logo for site as defined in Appearance or theme settings.
+* - site_name: Name for site as defined in Site information settings.
+* - site_slogan: Slogan for site as defined in Site information settings.
+*/
 #}
 {% block content %}
-<div class="display-flex margin-y-4">
+<a href="{{ path('<front>') }}" class="display-flex flex-align-center text-no-underline margin-y-4" rel="home">
   {% if site_logo %}
-    <a href="{{ path('<front>') }}" rel="home" class="site-logo display-block">
-      <img class="height-9" src="{{ site_logo }}" alt="{{ site_slogan}} {{ 'logo'|t }}" />
-    </a>
-  {% endif %}
-  <div class="site-info padding-x-2 padding-y-1">
-  {% if site_name %}
-    <div class="site-name">
-      <a href="{{ path('<front>') }}" class="font-heading-lg desktop:font-heading-xl text-primary text-no-underline text-bold" rel="home">{{ site_name }}</a>
-      <span class="usa-tag margin-left-1 bg-gold text-black">Beta</span>
-    </div>
-  {% endif %}
-  {% if site_slogan %}
-    <a class="site-slogan font-sans-2xs desktop:font-sans-xs text-base-dark text-no-underline hover:text-underline" href="https://www.noaa.gov/">{{ site_slogan }}</a>
+  <img class="height-9" src="{{ site_logo }}" alt=""/>
   {% endif %}
+  <div class="site-info padding-x-2 display-flex flex-column flex-align-center">
+    {% if site_name %}
+    <span class="site-name display-block font-heading-lg desktop:font-heading-xl text-primary text-bold">
+      {{ site_name }}
+    </span>
+    {% endif %}
+    {% if site_slogan %}
+    <span class="site-slogan display-block font-sans-2xs desktop:font-sans-xs text-base-dark">
```

Comment:
Very design nit pick, but to optically align the slogan with the site name could you add `padding-left-05`

Reply:
Addressed in 81cca8c